packages/graphql_flutter/pubspec.yaml

```diff
@@ -1,24 +1,31 @@
 name: graphql_flutter
 description: A GraphQL client for Flutter, bringing all the features from a modern
   GraphQL client to one easy to use package.
-version: 4.0.2-beta.1
+version: 4.0.2-beta.2
 homepage: https://github.com/zino-app/graphql-flutter/tree/master/packages/graphql_flutter
+
 dependencies:
-  graphql: ^4.0.2-beta.1
-  gql_exec: ^0.2.2
+  graphql: ^4.0.2-beta.2
+  gql_exec: ^0.3.0-nullsafety.1
   flutter:
     sdk: flutter
-  meta: ^1.1.6
+  meta: ^1.3.0
   path_provider: ^1.6.10
-  path: ^1.6.4
-  connectivity: ^2.0.0
+  path: ^1.8.0
+  connectivity: ^3.0.2
   hive: ^1.3.0
+
 dev_dependencies:
-  pedantic: ^1.8.0+1
+  pedantic: ^1.11.0
   mockito: ^4.0.0
   flutter_test:
     sdk: flutter
-  test: ^1.5.3
-  http: ^0.12.1
+  test: ^1.16.5
+  http: ^0.13.0
+
+dependency_overrides:
+  source_span: ^1.8.1
+  gql: ^0.13.0-nullsafety.1
+
 environment:
   sdk: '>=2.6.0 <3.0.0'
```

Comment:
Tests failed: It requires SDK version >=2.12.0-0 <3.0.0

Reply:
Is there any chance we can update the flutter version on CI?

Reply:
I thought it was automatic but apparently no

Reply:
@AndreHaueisen Can you try updating `cirrusci/flutter:latest-web` in .circleci file

Reply:
maybe cirrusci/flutter:stable